src/Microsoft.VisualStudio.Services.Agent/Util/VarUtil.cs

```diff
@@ -14,12 +16,12 @@ public static StringComparer EnvironmentVariableKeyComparer
         {
             get
             {
-                switch (Constants.Agent.Platform)
+                switch (PlatformUtil.RunningOnOS)
                 {
-                    case Constants.OSPlatform.Linux:
-                    case Constants.OSPlatform.OSX:
+                    case PlatformUtil.OS.Linux:
+                    case PlatformUtil.OS.OSX:
                         return StringComparer.Ordinal;
-                    case Constants.OSPlatform.Windows:
+                    case PlatformUtil.OS.Windows:
                         return StringComparer.OrdinalIgnoreCase;
```

Comment:
maybe we could use this method (or parallel one) for getting the comparer for the Dictionary<string,string> earlier in the review?

Reply:
Done.